pylon-core/src/main/kotlin/io/github/pylonmc/pylon/core/command/PylonCommand.kt

```diff
@@ -78,7 +78,7 @@ internal class PylonCommand : BaseCommand() {
     @Description("Toggle your WAILA bossbar")
     @CommandPermission("pylon.command.waila")
     fun waila(player: Player) {
```

Comment:
```suggestion
    fun toggleWaila(player: Player) {
```
naming(ik this is not part of this PR)